### ISSUE TITLE
Provide detailed data augmentaion setup or codes mentioned in paper?

### ISSUE THREAD

**Issue by LightningChan** [yinanhe/ForgeryNet#25]
Hi, I think you add lots of data augmentation to the validation set. This results in a large gap between the training and validation sets. This can lead to a situation where a model performs well, possibly simply due to a closer augmentation setup used in the validation set. It is very unfair when compared methods on the validation set. So, I suggest you provide detailed data augmentation setup or codes mentioned in the paper.

**Comment by yinanhe** [comment]
We used [albumentations](https://github.com/albumentations-team/albumentations).

**Comment by davide-coccomini** [comment]
Can you provide the exact augmentations used in the three setups?
